plugins/server/io.ktor/request-validation/2.0/manifest.ktor.yaml

```
@@ -0,0 +1,7 @@
+name: Request Validation
+description: Adds validation for incoming requests
+vcsLink: https://github.com/ktorio/ktor/tree/main/ktor-server/ktor-server-plugins/ktor-server-request-validation
```

Should we use a permanent link with the version?
```suggestion
vcsLink: https://github.com/ktorio/ktor/tree/3.0.0/ktor-server/ktor-server-plugins/ktor-server-request-validation
```